app/uk/gov/hmrc/crdlcache/models/CodeListSnapshot.scala

```diff
@@ -28,7 +29,7 @@ case class CodeListSnapshot(
 object CodeListSnapshot {
   def fromDpsSnapshot(
     config: CodeListConfig,
-    dpsSnapshot: dps.CodeListSnapshot
+    dpsSnapshot: codeList.CodeListSnapshot
```

Comment:
Hmm, one of the things I liked about the previous package structure is that we could import `uk.gov.hmrc.crdlcache.models.dps` and refer to the DPS models by `dps.Whatever`.

Maybe it's time for us to rename the DPS models which coincide with our internal models so that they have a `Dps` prefix on their names. What do you think?